README.md

```diff
@@ -102,6 +102,7 @@ make deploy
 ### Current issues with the development
 We're currently in the process of transitioning into this web client from Scratch's existing structure. As we transition, there are going to be some issues along the way that relate to how this client needs to interact with the existing infrastructure to work properly in production.
 
+#### FALLBACK ####
```

Comment:
@liam4 nice catch, though I would not say this is a matter of "bad" or "good", just convention. @LiFaytheGoblin would you mind removing the trailing `####`? Probably best to follow the other header conventions for now.